src/storage/write_ahead_log/disk_log_consumer_task.cpp

```diff
@@ -68,11 +68,16 @@ void DiskLogConsumerTask::DiskLogConsumerTaskLoop() {
 
   // Keeps track of how much data we've written to the log file since the last persist
   current_data_written_ = 0;
+  // Initialize sleep period
+  auto curr_sleep = persist_interval_;
+  auto next_sleep = curr_sleep;
+  const std::chrono::microseconds max_sleep = std::chrono::microseconds(10000);
   // Time since last log file persist
   auto last_persist = std::chrono::high_resolution_clock::now();
   // Disk log consumer task thread spins in this loop. When notified or periodically, we wake up and process serialized
   // buffers
   do {
+    //    curr_sleep = next_sleep;
```

Comment:
Remove unused code.

Reply:
Oops thats supposed to be commented in 😳